jest.config.js

```diff
@@ -34,6 +34,7 @@ module.exports = {
     '<rootDir>/packages/react-native/sdks',
     '<rootDir>/packages/react-native/Libraries/Renderer',
     '<rootDir>/packages/rn-tester/e2e',
+    '<rootDir>/packages/react-native-test-renderer/src',
```

Comment:
Needed to add this because the package has its own jest config and running from root fails. Is there a way to make sure these tests are run against the custom config in CI moving forward?